src/OpenTelemetry.Instrumentation.AspNetCore/AspNetCoreInstrumentationFilterResultType.cs

```diff
@@ -0,0 +1,34 @@
+// <copyright file="AspNetCoreInstrumentationFilterResultType.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenTelemetry
+{
+    /// <summary>
+    /// Represents request filtering result.
+    /// </summary>
+    public enum AspNetCoreInstrumentationFilterResultType
```

Comment:
@open-telemetry/dotnet-approvers - As discussed in SIG meeting, I have updated this. Let me know if there are any other name suggestions. Thanks!

Reply:
the ns already indicate this is AspNetCore...may be just `FilterResultType`. Waiting to hear other thoughts too.